flake.nix

```
@@ -0,0 +1,44 @@
+{
+  description = "faster shell navigation of projects";
+
+  inputs = {
+    nixpkgs.url = "https://flakehub.com/f/NixOS/nixpkgs/0.1.0";
```

```suggestion
    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

What does the 0.1.0 stand for in the context of nixpkgs? Shouldn't it be 23.11 or something similar?

I prefer to keep the canonical source of nixpkgs from the channels.

---

0.1.0 resolves to the current nixos-unstable.

If you'd like to track the latest stable, we could do this which always pulls the most current stable:

```suggestion
    nixpkgs.url = "https://flakehub.com/f/nixos/nixpkgs/*";
```

which currently resolves to `0.2311.556770+rev-8ac30a39abc5ea67037dfbf090d6e89f187c6e50`.

I of course don't mind using the upstream channel, but then to keep it from going really out of date without active maintenance, maybe using nixpkgs-unstable? What do you think?

---

I think I'll humour you with the flakehub changes for now. Let's see how it goes.

---

Let me know what you think! :)

---

Small thing: I wish the flakehub-push action would output a summary with a link to the release. Something like: "Done! rev XYX is now set as the release ABC at https://flakehub.com/f/zimbatm/h". As a new user who received a PR for my repo, it's not obvious where to go otherwise.